docs/attributes-registry/agent.md

```diff
@@ -0,0 +1,21 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Agent
+
+## Agent Attributes
+
+<!-- semconv registry.agent(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `agent.distro` | string | Agent distribution. [1] | `github.com/signalfx/splunk-otel-collector`; `github.com/aws-observability/aws-otel-collector` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `agent.id` | string | Unique identifier of agent instance. [2] | `627cc493-f310-47de-96bd-71410b7dec09` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
What's the difference with `service.instance.id` if any? Is this persistent across restarts?

Reply:
> What's the difference with service.instance.id if any? 

In the case when telemetry is being collected for the OTel Collector itself there's maybe no difference. 
However, the idea of `agent.*` fields in ECS is also to have the possibility attach that meta-information to telemetry data for a different, observed service.

For example, let's say the OTel collector collects metrics (or so) from an external service (let's say NGINX).
Then the telemetry data could contain:

- `service.name: NGINX`
- `service.instance.id: 123`
- `agent.type: io.opentelemetry.collector`
- `agent.id: xyz`

@tigrannajaryan not sure if this PR had also this kind of scenario in mind?

Reply:
>What's the difference with service.instance.id if any? 

It is the agent's instance id. Conceptually the same thing.

>Is this persistent across restarts?

Preferably it should, but I don't want to mandate that in semconv, since some agents may not have persistence capability. We can add recommendations.

>So with that we can differentiate the observer service from the observed service.

It was not a goal for me, but appears useful. When recording both of these attribute sets you essentially allow answering the question "who collected this telemetry?". My one concern would be that it bloats all telemetry coming from the Collector, so I am not sure if we should do it.